lib/alaveteli_localization.rb

```diff
@@ -33,21 +23,34 @@ def set_default_locale_urls(include_default_locale_in_urls)
         include_default_locale = include_default_locale_in_urls
     end
 
-    def set_session_locale(*args)
-      requested = args.compact.delete_if { |x| x.empty? }.first
+    # rubocop:disable Naming/AccessorMethodName
+    def set_default_locale(locale)
```

Comment:
There are no specs for this method but it's only ever used in `#with_default_locale` in `spec/spec_helper.rb`. Could refactor to remove this completely.